apps/src/music/player/MusicPlayer.js

```diff
@@ -255,12 +241,16 @@ export default class MusicPlayer {
     }
   }
 
-  convertSecondsToMeasure(seconds) {
-    return Math.floor(seconds / this.secondsPerMeasure());
+  // Converts actual seconds used by the audio system into a playhead
+  // position, which is 1-based and scaled to measures.
+  convertSecondsToPlayheadPosition(seconds) {
+    return 1 + seconds / this.secondsPerMeasure();
   }
 
-  convertMeasureToSeconds(measure) {
-    return this.secondsPerMeasure() * measure;
+  // Converts a playhead position, which is 1-based and scaled to measures,
+  // into actual seconds used by the audio system.
+  convertPlayheadPositionToSeconds(playheadPosition) {
+    return this.secondsPerMeasure() * (playheadPosition - 1);
```

Comment:
Nice 😎

Reply:
Thanks.  I feel like these two conversion functions are at the heart of this whole journey! :)